src/widget/controlwidgetconnection.cpp

```diff
@@ -124,7 +124,8 @@ void ControlWidgetPropertyConnection::slotControlValueChanged(double v) {
 
     if (!pWidget->setProperty(m_propertyName.constData(),parameter)) {
         qDebug() << "Setting property" << m_propertyName
```

Comment:
I guess this should become a warning because this behavior is unexpected? Maybe even enclosed in VERIFY_OR_DEBUG_ASSERT if it should not happen in a release version?

Reply:
Yes, this should be a warning. But it should be shown with in somewhat performant --developer mode already when designing skins for example.

Anyway, I didn't yet fully understand why the the warning is logged in the first place when the highlight connection actually works in the skin. I noticed this with several widgets in #2736 which work perfectly.